index.html

```diff
@@ -10,14 +10,11 @@
   specStatus: "ED",
   edDraftURI: "https://w3c.github.io/geolocation-api/",
   editors: [{
-    name: "Andrei Popescu",
-    company: "Google Inc.",
-    companyURL: "https://google.com/",
-    w3cid: "36565"
+    name: "Devices and Sensors Working Group"
   }],
-  wg: "Web Incubator Community Group",
-  wgURI: "https://wicg.io",
-  wgPublicList: "public-geolocation",
+  wg: "Devices and Sensors Group",
+  wgURI: "https://www.w3.org/das",
+  wgPublicList: "public-device-apis",
```

Comment:
Probably drop the mailing list. That will automatically make ReSpec tell people to file bugs on Github.

Reply:
Nice feature! Added in https://github.com/w3c/geolocation-api/pull/30/commits/84571110c1c05697b25edf9848a8f88ca6741b0e